frontend/src/models.ts

```diff
@@ -420,7 +420,7 @@ export type TimeFormatted = Time<string>;
 
 export type SelectOption = {
   label: string;
-  value: number;
+  value: number|string;
```

Comment:
tbh this should really just be a string, and whatever needs to consume value should parseInt it if needed.

Reply:
No problem, I can look into this. Just thought to keep it compatibel, but I totally agree.